source/shadow.c

```diff
@@ -552,11 +552,10 @@ ShadowStatus_t Shadow_GetTopicString( ShadowTopicStringType_t topicType,
         ( pOutLength == NULL ) )
     {
         shadowStatus = SHADOW_BAD_PARAMETER;
-        LogError( ( "Invalid input parameters pTopicBuffer: %p, pThingName: %p, thingNameLength: %u, topicType: %u, pOutLength: %p",
-                    ( void * ) pTopicBuffer,
-                    ( void * ) pThingName,
-                    thingNameLength,
-                    topicType,
+        LogError( ( "Invalid input parameters pTopicBuffer: %p, pThingName: %p, "
+                    "thingNameLength: %u, topicType: %d, pOutLength: %p.",
+                    ( void * ) pTopicBuffer, ( void * ) pThingName,
```

Comment:
I personally like the presence of parameters on separate line, but that's just my style preference.